process_pr.py

```diff
@@ -684,6 +687,10 @@ def process_pr(repo_config, gh, repo, issue, dryRun, cmsbuild_user=None, force=F
         signing_categories.add("tests")
         push_test_issue = True
     except: pass
+    if (not issue.pull_request) and re.match(CREATE_REPO, issue.title):
+      with open("query-new-data-repo-issues.properties", "w") as f:
```

Comment:
add the issue number in the property file name so that for each issue it creates a unique file